DOS build no longer works

### Operating System

Other (please specify)

### DevilutionX version

Custom build (please specify commit ID)

### Describe

I have working environment, where I'm building DOS, 98 and XP builds (for testing) of DevilutionX, and I found out now, that DOS build no longer works. When I try to compile latest source code, I'm getting this error right at the end of building process:

```[100%] Built target libdevilutionx
[100%] Building CXX object CMakeFiles/devx.dir/Source/main.cpp.obj
[100%] Linking CXX executable devx
/opt/i386-pc-msdosdjgpp-toolchain/lib/gcc/i386-pc-msdosdjgpp/14.2.0/../../../../i386-pc-msdosdjgpp/bin/ld: Source/CMakeFiles/libdevilutionx.dir/diablo.cpp.obj:diablo.cpp:(.text._ZN10devilution12_GLOBAL__N_125OptionLanguageCodeChangedEv+0x1b): undefined reference to `devilution::effects_cleanup_sfx(bool)'
/opt/i386-pc-msdosdjgpp-toolchain/lib/gcc/i386-pc-msdosdjgpp/14.2.0/../../../../i386-pc-msdosdjgpp/bin/ld: Source/CMakeFiles/libdevilutionx.dir/diablo.cpp.obj:diablo.cpp:(.text._ZN10devilution11diablo_quitEi+0x82): undefined reference to `devilution::effects_cleanup_sfx(bool)'
/opt/i386-pc-msdosdjgpp-toolchain/lib/gcc/i386-pc-msdosdjgpp/14.2.0/../../../../i386-pc-msdosdjgpp/bin/ld: Source/CMakeFiles/libdevilutionx.dir/diablo.cpp.obj:diablo.cpp:(.text._ZN10devilution10DiabloMainEiPPc+0x101b): undefined reference to `devilution::effects_cleanup_sfx(bool)'
/opt/i386-pc-msdosdjgpp-toolchain/lib/gcc/i386-pc-msdosdjgpp/14.2.0/../../../../i386-pc-msdosdjgpp/bin/ld: Source/CMakeFiles/libdevilutionx.dir/lua/lua_global.cpp.obj:lua_global.cpp:(.text._ZN10devilution19LuaReloadActiveModsEv+0x55a): undefined reference to `devilution::effects_cleanup_sfx(bool)'
collect2: error: ld returned 1 exit status
gmake[2]: *** [CMakeFiles/devx.dir/build.make:550: devx] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:936: CMakeFiles/devx.dir/all] Error 2
gmake: *** [Makefile:156: all] Error 2
```

I did look for problematic commit (because I know, that I was able to compile code previously without issues), and found out, that the error was introduced in following commit c55e7a4107ccb341c8eb7a92e892fa11dbb8e62b. Previous commit (714ae37fbaf89f2700e759b7a13561ed6ea12908) compiled without error. Other versions (Windows) are unaffected.

### To Reproduce

Build latest source code for DOS with commands provided in Build from Source article on wiki.

### Expected Behavior

_No response_

### Additional context

_No response_

--- a/Source/effects_stubs.cpp
+++ b/Source/effects_stubs.cpp
@@ -44,7 +44,7 @@ void PlaySfxLoc(SfxID psfx, Point position, bool randomizeByCategory)
 }
 void sound_stop() { }
 void sound_update() { }
-void effects_cleanup_sfx() { }
+void effects_cleanup_sfx(bool fullUnload) { }
 void sound_init() { }
 void ui_sound_init() { }
 void effects_play_sound(SfxID id) { }

